pandas/core/tools/numeric.py

```diff
@@ -39,7 +39,7 @@ def to_numeric(arg, errors="raise", downcast=None):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
-    downcast : {'integer', 'signed', 'unsigned', 'float'} , default None
+    downcast : {'int', 'signed', 'unsigned', 'float'} , default None
```

Comment:
This shouldn't change. probably error in validation. @datapythonista

Reply:
agree, this seems to refer to the string `integer`, I guess the function does expect `integer` and not `int`. But if you can remove the space before the comma that would be great.